src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarTownyChatEventListener.java

```diff
@@ -1,33 +1,85 @@
 package com.gmail.goosius.siegewar.listeners;
 
+import com.avaje.ebeaninternal.server.el.ElSetValue;
```

Comment:
Fairly certain this isn't a needed import

Reply:
Fixed